README.md

```diff
@@ -56,14 +56,13 @@ _**`❍ truffle develop> test`**_
 
 
 ```
-  Contract: Diesel Price Tests
+  Contract: ETH Price Tests
     ✓ Should get contract instantiation for listening to events
     ✓ Should have logged a new Provable query
-    ✓ Callback should have logged a new diesel price (4676ms)
-    ✓ Should set diesel price correctly in contract
+    ✓ Callback should have logged a new ETH price (4676ms)
+    ✓ Should set ETH price correctly in contract
     ✓ Should revert on second query attempt due to lack of funds (120ms)
 
```

Comment:
```suggestion
✓ Should succeed on a second query attempt when sending funds (97ms)
```

Reply:
Oops, forgot I added a new test & didn't update the `README` - thanks!